### ISSUE TITLE
Not loading in Redactor

### ISSUE THREAD

**Issue by ghost** [nystudio107/craft3-richvariables#44]
### Describe the bug
Quite simply, the Rich Variables icon won't appear in Redactor

### To reproduce
Steps to reproduce the behaviour:
1. Install plugin
3. Add plugin to Redactor config
4. Plugin does not appear
5. Console shows the following error:
```
Failed to load resource: the server responded with a status of 500 ()
init @ richvariables.js:69
```

Redactor config:
```
{
  "buttons": ["html","bold", "italic"],
  "plugins": ["richvariables"]
}
```

### Versions
- Plugin version: 4.0.0-beta.3
- Craft version: Craft Pro 4.0.3

**Comment by khalwat** [comment]
Can you give me more detail on this? I have Rich Variables up and running in my Craft 4 testing instance without issue.

Can you check your Craft `web.log` to give me details on what the `500` error is, specifically?

**Comment by khalwat** [comment]
Fixed in 4.0.0
